docs/source/examples/petstore/petstore.all.seal

```diff
@@ -1,3 +1,4 @@
+allow subject group banned to manage petstore.* where subject.iss == "";
```

Comment:
please use something more relevant:
```
# allow to inspect petstore.pet where subject.aud == "sysadmins";
```
the above would be valid, but subject is not optional until #59 is implemented.

```suggestion
deny subject group everyone to manage petstore.pet where subject.iss != "petstore.com";
```